src/js/static-filtering-parser.js

```diff
@@ -2806,81 +2805,81 @@ Parser.regexUtils = Parser.prototype.regexUtils = (( ) => {
 
     const toTokenizableStr = node => {
         switch ( node.type ) {
-            case 1: /* T_SEQUENCE, 'Sequence' */ {
```

Comment:
This entire block is now unindented by one level, and that's the only change here.

I picked this style based on `src/js/static-net-filtering.js`. We would have to pick one of the two, and this one is the default for ESLint.

Correct:

```js
switch (foo) {
case 1:
    console.log('foo is 1');
default:
    console.log('foo is none');
}
```

Incorrect:

```js
switch (foo) {
    case 1:
        console.log('foo is 1');
    default:
        console.log('foo is none');
}
```

Reply:
I looked at your changes for this case as a whole and I guess it's not that confusing, so I am fine with this one.

Reply:
Thanks for the explanation.

Unfortunately it seems like there's no way to turn off indentation rules for `switch...case` in general. There's [no "off" option](https://eslint.org/docs/rules/indent).

Maybe we can leave the indentation here as it is and let the warnings appear until we find a better way?

Alternatively we could use the style in this file and update `src/js/static-net-filtering.js` instead.